matlab/library/WBToolboxLibrary_repository.mdl

```diff
@@ -5934,12 +5932,94 @@ Library {
 	    }
 	  }
 	}
+	Block {
+	  BlockType		  S-Function
+	  Name			  "YarpRpc"
+	  SID			  "1848"
+	  Ports			  [1]
+	  Position		  [-70, 105, 50, 135]
+	  ZOrder		  252
+	  FunctionName		  "BlockFactory"
+	  Parameters		  "'YarpRpc','WBToolbox',portName,rpcCommand"
+	  SFunctionDeploymentMode off
+	  EnableBusSupport	  off
+	  SFcnIsStateOwnerBlock	  off
+	  Object {
+	    $PropName		    "MaskObject"
+	    $ObjectID		    296
+	    $ClassName		    "Simulink.Mask"
+	    Type		    "YARP Rpc"
+	    Description		    "This block behaves as the command 'yarp rpc'. \n\nIt takes two parameters:\n1) Rpc server port "
+	    "name to which user wants to send rpc commands. This is a string datatype.\n2) Rpc command which is the command us"
+	    "er wants to send. This is a string datatype.\n\nThis block has an input signal called Trigger. This is intended t"
+	    "o be a boolean datatype and when the signal is true the rpc command is sent to the rpc server port."
```

Comment:
I would explicitly specify that the block is **blocking the simulation** until a reply is received, as one could expect that a separate thread is spawned to handle the RPC communication.

Reply:
Done